evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs

```diff
@@ -202,7 +203,7 @@ impl<'a, B: Backend> Executor<CheatcodeStackState<'a, B>, CheatcodeStackExecutor
         // Need to create a non-empty contract at the cheat code address so that the EVM backend
         // thinks that something exists there.
         evm.initialize_contracts([(*CHEATCODE_ADDRESS, vec![0u8; 1].into())]);
-
+        evm.initialize_contracts([(*CONSOLE_LOG_ADDRESS, vec![0u8; 1].into())]);
```

Comment:
Make this part of the previous call since it receives an iterator of address/bytes